openebench/APAeval_schemas/Challenges/APAeval_challenge1.json

```diff
@@ -0,0 +1,38 @@
+{
+    "_id": "APAeval:2021-06-04_Q2",
+    "_schema": "https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/Challenge",
+    "name": "Correlation benchmark for RNA-seq poly(A) site quantification.",
+    "acronym": "Q2",
+    "benchmarking_event_id": "APAeval:2021-06-04",
+    "is_automated": true,
```

Comment:
Shouldn't this be `false`?

Reply:
for the moment doesn't matter, this is only a flag for us